packages/dcos-integration-test/extra/test_endpoints.py

```diff
@@ -144,6 +146,50 @@ def test_if_we_have_capabilities(cluster):
     assert {'name': 'PACKAGE_MANAGEMENT'} in r.json()['capabilities']
 
 
+def test_cosmos_package_add(cluster):
```

Comment:
@mellenburg is going to need to review these tests. I'm doing a first pass here.